skymp5-server/cpp/unit/DistContentsTest.h

```diff
@@ -0,0 +1,217 @@
+    "client\\Data\\SKSE\\Plugins\\version-1-5-62-0.bin",
+    "client\\Data\\SKSE\\Plugins\\version-1-5-73-0.bin",
+    "client\\Data\\SKSE\\Plugins\\version-1-5-80-0.bin",
+    "client\\Data\\SKSE\\Plugins\\version-1-5-97-0.bin",
+    "server\\ChakraCore.dll",
+    "server\\data\\scripts\\GM_Chat.pex",
+    "server\\data\\scripts\\GM_ChatAction.pex",
+    "server\\data\\scripts\\GM_ChatBind.pex",
+    "server\\data\\scripts\\GM_ChatNonRp.pex",
+    "server\\data\\scripts\\GM_ChatRp.pex",
+    "server\\data\\scripts\\GM_ChatSend.pex",
+    "server\\data\\scripts\\GM_Colors.pex",
+    "server\\data\\scripts\\GM_CommandBan.pex",
+    "server\\data\\scripts\\GM_Commands.pex",
+    "server\\data\\scripts\\GM_Distances.pex",
+    "server\\data\\scripts\\GM_Main.pex",
+    "server\\data\\scripts\\GM_String.pex",
+    "server\\data\\scripts\\M.pex",
+    "server\\data\\ui\\00e4a2200c3e5ae71d67e16d7990c68f.ttf",
+    "server\\data\\ui\\0935e00bec58845a687af990665e1554.png",
+    "server\\data\\ui\\268a0df3f0e2c23d1ea6e0e024316b34.ttf",
+    "server\\data\\ui\\6f802bf60c545304b09e5fed8484f5c7.ttf",
+    "server\\data\\ui\\build.js",
+    "server\\data\\ui\\fdf086661e1600f95a34561c377e0a9e.ttf",
+    "server\\data\\ui\\index.html",
+    "server\\dist_back\\chat.js",
+    "server\\dist_back\\chat.js.map",
+    "server\\dist_back\\index.js",
+    "server\\dist_back\\index.js.map",
+    "server\\dist_back\\libkey.js",
+    "server\\dist_back\\libkey.js.map",
+    "server\\dist_back\\manifestGen.js",
+    "server\\dist_back\\manifestGen.js.map",
+    "server\\dist_back\\nativeGameServer.js",
+    "server\\dist_back\\nativeGameServer.js.map",
+    "server\\dist_back\\publicIp.js",
+    "server\\dist_back\\publicIp.js.map",
+    "server\\dist_back\\scampNative.js",
+    "server\\dist_back\\scampNative.js.map",
+    "server\\dist_back\\serverInterface.js",
+    "server\\dist_back\\serverInterface.js.map",
+    "server\\dist_back\\settings.js",
+    "server\\dist_back\\settings.js.map",
+    "server\\dist_back\\systems\\clientVerify.js",
+    "server\\dist_back\\systems\\clientVerify.js.map",
+    "server\\dist_back\\systems\\login.js",
+    "server\\dist_back\\systems\\login.js.map",
+    "server\\dist_back\\systems\\masterClient.js",
+    "server\\dist_back\\systems\\masterClient.js.map",
+    "server\\dist_back\\systems\\spawn.js",
+    "server\\dist_back\\systems\\spawn.js.map",
+    "server\\dist_back\\systems\\system.js",
+    "server\\dist_back\\systems\\system.js.map",
+    "server\\dist_back\\ui.js",
+    "server\\dist_back\\ui.js.map",
+    "server\\dist_front\\skymp5-client.js",
+    "server\\gamemode.js",
+    "server\\launch_server.bat",
+    "server\\scamp_native.node",
+    "server\\server-settings.json"
+  };
+
+  size_t totalMissing = 0;
+  totalMissing += PrintMissing(paths, expectedPaths, "paths", "expectedPaths");
+  totalMissing += PrintMissing(expectedPaths, paths, "expectedPaths", "paths");
```

Comment:
An idea for future: we can run `diff` here, which will enable us to render a fancier missing/extra paths.